cinema/views.py

```diff
@@ -0,0 +1,43 @@
+from django.core.exceptions import ObjectDoesNotExist
+from django.shortcuts import get_object_or_404
+from rest_framework.response import Response
+from rest_framework.decorators import api_view
+from rest_framework import status
+
+
+from cinema.models import Movie
+from cinema.serializers import MovieSerializer
+
+
+def create_movie_or_raise(data, instance=None):
```

Comment:
The name of the method does not correlate with its functionality, rename it to `serialize_movie` or other